Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM julia:1.4
+FROM julia:1.5
```

Comment:
There are some other changes required for this to run on Julia 1.5 so let's skip this upgrade for now.